.circleci/config.yml

```diff
@@ -246,32 +204,31 @@ jobs:
       - restore_cache:
           name: Restoring GOCACHE
           keys:
-            - influxdb-gocache-{{ .Branch }}-{{ .Revision }} # Matches when retrying a single run.
-            - influxdb-gocache-{{ .Branch }}- # Matches a new commit on an existing branch.
-            - influxdb-gocache- # Matches a new branch.
+            - influxdb-gotest-{{ .Branch }}-{{ .Revision }} # Matches when retrying a single run.
+            - influxdb-gotest-{{ .Branch }}- # Matches a new commit on an existing branch.
+            - influxdb-gotest- # Matches a new branch.
```

Comment:
Trying to:
1. Bust the caches that are currently in Circle, to see if they rebuild smaller & quicker to download
2. Separate the test caches for different jobs to avoid collisions